deploy/deploy.py

```diff
@@ -209,6 +272,7 @@ def execute(
     deployment: str = "prod",
     install: bool = True,
     skip_package: bool = False,
+    sundeck_db: str = "SUNDECK",
```

Comment:
i thikn if sundeck_Db is `None` the installation of hte share shoudl be skipped (smoothly)

Reply:
np. Is the creation of a dummy view as this PR is currently doing sufficient? I could also push down a check into the setup script which detects if the app package has the view we expect and avoid creating the REPORTING view.

Reply:
if `sundeck_db is None`, we will run a trimmed down view creation that has no data so the setup script can be just assume the `global_query_history` view is always present.